src/components/schema/jsonld.tsx

```diff
@@ -0,0 +1,36 @@
+import {ClothingStore, LocalBusiness, Organization, Thing, WithContext, PostalAddress} from 'schema-dts';
+
+export function JsonLd<T extends Thing>(json: WithContext<T>): string {
+  return `<script type="application/ld+json">
```

Comment:
Does this need to be a string or could it be a jsx `<script>` element?

Reply:
It can be a a script element